devtool

```diff
@@ -73,6 +72,7 @@ install_package(){
 all() {
     lint
     test_with_coverage
+    integ_tests
```

Comment:
Suggestion: exclude devtool changes from this PR. As a workaround you may add a section to README and state clearly that internal contributors must run "devtool integ_tests".

Reason: currently integ_tests download data from S3 bucket s3://sagemaker-clarify-datasets which is not publicly available. So it is excluded from all() on purpose so that everyone can run through the setup steps in README.

Reply:
I checked the example notebooks. They do not use any s3 bucket. They download data from either sagemaker-sample data, or UCI database. Besides, these example notebooks were already running in devtool under `test_with_coverage`. I just moved it to `integ_tests`.

Reply:
Approving this pr since the examples don't use only public dataset, https s3 url which is public

Reply:
For tracking: addressed by https://github.com/aws/amazon-sagemaker-clarify/pull/130